### PR TITLE
.werft/observability: Add stackdriver support

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

Adds support for Grafana's stackdriver datasource 

## Related Issue(s)

Related to https://github.com/gitpod-io/gitpod/issues/7859

## How to test

Deploy and access grafana following the [How-to](https://www.notion.so/gitpod/Getting-access-to-Grafana-and-Prometheus-in-preview-environments-f2938b2bcb0c4c8c99afe1d2b872380e), try to create a new dashboard using the stackdrive datasource. It should work :) 

image

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **arthursens** and additionally assign **mustard-mh** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mustard-mh` in a comment when ready.

Associated issue: *#7859*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[.werft/observability/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/.werft/observability/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by wulfthimm — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 9a2b4ca461db7d721bef7bdb1f2971464258999e</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7954#pullrequestreview-870151273" title="Approved">wulfthimm</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mustard-mh** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mustard-mh` in a comment when ready.

Associated issue: *#7859*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- ~~[.werft/observability/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/.werft/observability/OWNERS)~~ [wulfthimm]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7954#pullrequestreview-870151273" title="Approved">wulfthimm</a>*

Associated issue: *#7859*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- ~~[.werft/observability/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/.werft/observability/OWNERS)~~ [wulfthimm]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
